ocf_data_sampler/load/load_dataset.py

```diff
@@ -0,0 +1,55 @@
+""" Loads all data sources """
```

Comment:
Oh this is nice, I like that this is moved out of datasets!

Reply:
thanks